inlong-manager/manager-common/src/main/java/org/apache/inlong/manager/common/util/AESUtils.java

```diff
@@ -58,8 +58,9 @@ public class AESUtils {
     private static Properties getApplicationProperties() throws IOException {
         Properties properties = new Properties();
         String path = Thread.currentThread().getContextClassLoader().getResource("").getPath() + CONFIG_FILE;
-        InputStream inputStream = new BufferedInputStream(Files.newInputStream(Paths.get(path)));
-        properties.load(inputStream);
+        try (InputStream inputStream = new BufferedInputStream(Files.newInputStream(Paths.get(path)))) {
```

Comment:
Hello @TYzzt, thanks for your contribution.

Is there any other similar code that needs to be modified?

Reply:
two modifications have been added. not everyone resource should be close with try-with-resource